packages/flutter_test/lib/src/_matchers_io.dart

```diff
@@ -66,6 +69,7 @@ class MatchesGoldenFile extends AsyncMatcher {
       throw 'must provide a Finder, Image, or Future<Image>';
     }
 
+    goldenFileComparator.setPrecisionTolerance(precisionTolerance);
```

Comment:
Instead of modifying the global state of the goldenFileComparator (which could be a surprising side effect), could the tolerance just be passed in below to the compare method?

Reply:
I thought of going that route, but it might be more breaking that way. Still evaluating. The comparator class (and therefore `compare` function) is extended/implemented/overridden by many customers.